src/lib/components/SearchBar/QueryBuildHelper.js

```diff
@@ -1,18 +1,23 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { List, Grid } from 'semantic-ui-react';
+import { withState } from 'react-searchkit';
 
-export class QueryBuildHelper extends Component {
+class _QueryBuildHelper extends Component {
```

Comment:
This is because we export it as `QueryBuildHelper` wrapped `withState` in last line to get access to RSK https://github.com/inveniosoftware/react-invenio-app-ils/pull/233/files/453978f7eea142005a8454640d7db561e2472437#diff-0085bd5ffc13ff97a4d69cbc5f2d54f8R73